addon/components/tool-tipster.js

```diff
@@ -84,8 +84,8 @@ export default Ember.Component.extend({
       }
     });
     options.trigger = this.get('triggerEvent');
-    // Handle SafeString
-    if (content instanceof SafeString) {
+    // Handle htmlSafe
+    if (content instanceof htmlSafe) {
       options.content = content.toString();
```

Comment:
Maybe we can use this polyfill https://github.com/workmanw/ember-string-ishtmlsafe-polyfill to maintain backwards compatibility.